packages/design-system/src/components/Navigations/IconTab/Tab.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
```

Comment:
React 17부터 [jsx transform](https://ko.reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html) 기능이 내장되면서 React를 import 하지 않아도 jsx 파싱을 지원해줍니다.
다만 emotion 사용시 jsx pragma가 필요한 이슈가 있어 다음과 같은 주석이 추가되어야 합니다.
```suggestion
/** @jsxImportSource @emotion/react */
```
jsx pragma를 제거할 수도 있는데, 관련 바벨 설정을 건드리다가 뭔가 잘 안되어서 나중에 다시 살펴보려고 합니다. 우선은 번거롭지만 해당 주석을 추가해주세요.🥲